### DISCUSSION TITLE
Wake up the car?

### DISCUSSION THREAD

**Discussion by andreha2023** [tillsteinbach/WeConnect-mqtt#234] [Q&A] [answered]
Hello,
how can i wake up my passat gte?

i have the following writeable topics:

vwpassat/vehicles/ID/controls/wakeup
vwpassat/vehicles/ID/controls/wakeup_writetopic

Thank you!! 
André

**Comment by tillsteinbach** [comment, marked as answer]
Try to send the string `start` to the topic `vwpassat/vehicles/ID/controls/wakeup_writetopic`

**Comment by andreha2023** [comment]
Thanks!! That works fine!
